app/src/lib/stores/app-store.ts

```diff
@@ -5091,7 +5142,7 @@ export class AppStore extends TypedBaseStore<IAppState> {
     return Promise.resolve()
   }
 
-  public async _resolveCurrentEditor() {
+  public _resolveCurrentEditor = async () => {
```

Comment:
making this change so we can pass this into the TutorialAssessor without worrying about `this` binding issues